<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_pose">
     <property name="text">
      <string>   Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_connect">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_disconnect">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_sleep">
     <property name="text">
      <string>Deep Sleep</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
